index.html

```diff
@@ -1,234 +1,253 @@
+        <p class="work-details prodetails">Tribute to Dr. Abdul Salam a Nobel Scientist.</p>
+        <div class="flex pro2flex">
+          <p class="flex-item item1 proitem1">css</p>
+          <p class="flex-item item2 proitem2">html</p>
+          <p class="flex-item item3 proitem3">bootstrap</p>
+        </div>
+      </div>
+      <button class="work-button probutton">See Project</button>
+    </section>
+    <section id="aboutme">
+      <h2 id="me">About me</h2>
+      <p>
+        I can help you to build a product, feature or website Look through
+        some of my work and experience! If you like what you see and have a
+        project you need coded, don't hesitate to contact me.
+      </p>
+      <button class="work-button resumebtn">Get My Resume</button>
+      <div><img id="aboutmeimg" src="/img/illustration-about-me-mobile.png" alt=""></div>
+    </section>
+    <section id="languages">
+      <div class="thumbnail">
+        <div class="rectangle">
+        </div>
+        <h2 class="headings">Languages</h2>
+      </div>
+      <ul>
+        <ul class="marginlistitems">
+          <li class="listitems">Javascript</li>
+          <li class="listitems">Ruby</li>
+          <li class="listitems">Html</li>
+          <li class="listitems">Css</li>
+        </ul>
+        <div class="thumbnail">
+          <div class="rectangle square">
+          </div>
+          <h2 class="headings">Frameworks</h2>
         </div>
-    </footer>
-  </body>
-</html>
+        <ul class="marginlistitems">
+          <li class="listitems">Bootstrap</li>
+          <li class="listitems">Ruby on Rails</li>
+          <li class="listitems">RSPec</li>
+          <li class="listitems">CapyBara</li>
+          <li class="listitems">Selenium</li>
+        </ul>
+        <div class="thumbnail">
+          <div class="rectangle circle">
+          </div>
+          <h2 class="headings ">Skills</h2>
+        </div>
+        <ul class="marginlistitems">
+          <li class="listitems ">Codekit</li>
+          <li class="listitems">Github</li>
+          <li class="listitems">Codepen</li>
+          <li class="listitems">Gitlab</li>
+          <li class="listitems">Terminal</li>
+        </ul>
+      </ul>
+    </section>
+
+    <form action="https://formspree.io/f/xoqzdnpa"
+    method="POST">
+      <section id="getintouch" class="border">
+        <h2 class="intouch">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch</h2>
+        <input class="input border" type="text" placeholder="Full name" maxlength="30" required>
+        <input class="input border" type="email" placeholder="Email address" required>
+        <textarea class="input" name="textarea" id="textarea" cols="30" rows="5" placeholder="Enter text here"
+          maxlength="500" required></textarea>
+          <div class="centerbtn"><button type="submit" class="getintouchbtn work-button">Get in touch</button>
```

Comment:
- [ ] Kindly consider applying the `pressed` state in button interactions as suggested in the figma design by using the `:active` pseudoclass as our design need to be similar to the figma design.
figma design
![er](https://user-images.githubusercontent.com/96683843/220424376-8de8a685-2a2c-4fe4-9f23-928311129856.jpg)